src/Apps/Collect2/Routes/Collect/index.tsx

```diff
@@ -40,16 +41,18 @@ export const CollectApp = track({
   const medium = params && params.medium
   const { description, breadcrumbTitle, title } = getMetadataForMedium(medium)
   const { trackEvent } = useTracking()
+  const { user } = useSystemContext()
 
   const canonicalHref = medium
     ? `${sd.APP_URL}/collect/${medium}`
     : `${sd.APP_URL}/collect`
 
   // Client renders will get COLLECTION_HUBS from sd; server renders
   // will get it from the SystemContext.
-  const showCollectionHubs =
-    sd.COLLECTION_HUBS === "experiment" ||
-    props.COLLECTION_HUBS === "experiment"
+  const isExperiment =
+    sd.COLLECTION_HUB_ENTRYPOINTS === "experiment" ||
+    props.COLLECTION_HUB_ENTRYPOINTS === "experiment"
+  const showCollectionHubs = isExperiment && !user
```

Comment:
Do I understand correctly: we are showing the hubs nav to [50% of logged in users](https://github.com/artsy/force/pull/4582/files#diff-87a40dff132ddf1754949c639c8da872R31-R32) and 100% of non-logged-in users?

Reply:
I got this mixed up with the homepage entry points. We should show these entry points at all times.